recipes/sigmut/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sigmut" %}
+{% set version = "v1.0" %}
```

Comment:
Our version numbers don't contain `v`

Reply:
I've seen some other recipes using 'v' for versions. Anyhow, thanks again for your suggestion; I will modified the file and try the PR again.